library/proc_macro/src/lib.rs

```diff
@@ -847,8 +847,8 @@ pub struct Ident(bridge::client::Ident);
 impl Ident {
     /// Creates a new `Ident` with the given `string` as well as the specified
     /// `span`.
-    /// The `string` argument must be a valid identifier permitted by the
-    /// language, otherwise the function will panic.
+    /// The `string` argument must be either a valid identifier permitted by the
+    /// language, or a keyword (e.g. `self`). Otherwise, the function will panic.
```

Comment:
``must be either a valid identifier permitted by the language, or a keyword (e.g. `self`)``
->
``must be a valid identifier permitted by the language (that includes keywords, e.g. `self`)``
Keywords are a subset of identifiers in the Rust model.